cmd/root.go

```diff
@@ -74,15 +80,64 @@ func run(cmd *cobra.Command, args []string) {
 		}
 	}()
 
+	id, err := os.Hostname()
```

Comment:
Does this assume that each pod runs on a different node? Or will the pod see a different hostname?

Reply:
Within a cluster it actually gets the name of the pod when calling `os.Hostname()`

Some logging output from my testing:
```
astro-6dcc6474-6985q astro time="2019-12-04T19:37:28Z" level=info msg="astro-6dcc6474-ddpw2 is now the leader"
astro-6dcc6474-lt7k5 astro time="2019-12-04T19:37:28Z" level=info msg="astro-6dcc6474-ddpw2 is now the leader"
```